easybuild/easyconfigs/k/kalign2/kalign2-3.3.1-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,32 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'kalign2'
+version = '3.3.1'
+
+homepage = 'https://github.com/TimoLassmann/kalign'
+description = "Kalign is a fast multiple sequence alignment program for biological sequences."
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/TimoLassmann/kalign/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['7f10acf9a3fa15deabbc0304e7c14efa25cea39108318c9f02b47257de2d7390']  # 3.3.1.tar.gz
+
+builddependencies = [('binutils', '2.35')]
+
+dependencies = [
+    ('Autoconf', '2.69'),
+]
```

Comment:
I just realized this now, but this is a builddep. Additionally, I think we should prefer the Autotools bundle instead of just Autoconf

Reply:
Good idea. I changed that over.

Reply:
You didn't change it to a builddep

Reply:
I did not think. Done now, sorry!